app/templates/direct-award/view-project.html

```diff
@@ -46,7 +46,12 @@
             "title": "Write a list of your requirements",
             "content": [
               {"type": "text", "text": "Work with buyers, someone who will use the service and technical experts to prepare a list of ‘must-haves’ and ‘wants’. These should inform your search category, keywords and filters."},
-              {"type": "text", "text": "You can also <a href=\"https://www.gov.uk/guidance/talking-to-suppliers-before-you-buy-digital-marketplace-services\" target=\"_blank\" rel=\"external noopener noreferrer\">talk to suppliers before you start</a> to help you refine your requirements."|safe}
+              {"type": "text", "text": "You can also <a href=\"https://www.gov.uk/guidance/talking-to-suppliers-before-you-buy-digital-marketplace-services\" target=\"_blank\" rel=\"external noopener noreferrer\" 
+                data-analytics=\"trackEvent\"
+                data-analytics-category=\"Direct Award\"
+                data-analytics-action=\"External Link\"
+                data-analytics-label=\"https://www.gov.uk/guidance/talking-to-suppliers-before-you-buy-digital-marketplace-services\"
```

Comment:
Feels like if we have some JavaScript automating this process, then it ought to be able to grab the URL automatically, rather than have to duplicate it into the label field each time?

Reply:
I could improve the JS by adding additional functionality such as pulling the link or text if the `data-analytics-label` is missing